### ISSUE TITLE
Target folder issue

### ISSUE THREAD

**Issue by MJ4Dev** [muzammilkm/Nahl.AzurePipeline.Tasks#6]
The target folder I have is  something like 'Dept/A Reports' under the report server, which already exist, but I keep getting the error about unable to create the target folder ( that actually is correct because we don't have the permission to create or delete folders), so I'm not sure how to get around this, would you help?

**Comment by muzammilkm** [comment]
Do you have permission to upload the report files on that folder ?

**Comment by MJ4Dev** [comment]
Yes we use service account to do that, to migrate from TFS to DevOps server. I tried to modify the script (SSRSReportsDeployment.ps1) to not create folder then now I got error saying I can't create reportDataSource, as all of that are already in place.

**Comment by MJ4Dev** [comment]
The connection should work as I see this in the log: 

Connected to https://_rptsvrurl_/reportserver/ReportService2010.asmx using <domain\svcAcct>

**Comment by muzammilkm** [comment]
your user <domain\svcAcct> should be assigned with ["Report Builder","Publisher"] minimum roles.
